lib/session/call-session.js

```diff
@@ -1883,7 +1883,7 @@ Duration=${duration} `
   wrapDialog(dlg) {
     dlg.connectTime = moment();
     const origDestroy = dlg.destroy.bind(dlg);
-    dlg.destroy = () => {
+    dlg.destroy = (opts, callback) => {
```

Comment:
I dont think we should include `callback`.  Its optional, if the caller wants a callback rather than a Promise.  But we exclusively use Promises